core/platform/auth/firebase_auth_services_test.py

```
@@ -1781,3 +1777,347 @@ def test_delete_external_auth_associations_when_get_users_raise_error(
+            ),
+        ):
+            firebase_auth_services.delete_multi_external_accounts(accounts)
+
+    def test_delete_accounts_with_none_auth_id(self) -> None:
+        self.firebase_sdk_stub.create_user('uid_0')
+        accounts = [
+            auth_domain.ExternalAccount(auth_id=None),
+            auth_domain.ExternalAccount(auth_id='uid_0'),
+        ]
+
+        with self.assertRaisesRegexp(
```

Ruff (0.14.14)

[warning] `assertRaisesRegexp` is deprecated, use `assertRaisesRegex`

Replace `assertRaisesRegex` with `assertRaisesRegexp`

(UP005)